### PR TITLE
RestoreSnapshotModal - ch29948/restore-modal-requests-app-slug-which-may

### PR DESCRIPTION
**Pull request by abhiaiyer91**
Added clarifying copy in RestoreSnapshotModal. 

I originally was trying to get to this modal organically using the product, but overall my speed to add this change was pretty slow. I've added storybook to the web project so we can iterate on stateless components faster.

image

### CODE REVIEW

**Comment by AmberAlston** [comment]
@jgruica or @GraysonNull Can one of you review this PR please? Thanks!

**Review by jgruica — APPROVED** [review]

File: kotsadm/web/src/components/modals/RestoreSnapshotModal.jsx [hunk 1]
```diff
@@ -35,7 +35,9 @@ export default function RestoreSnapshotModal(props) {
             <p className="u-fontSize--normal u-fontWeight--normal u-color--dustyGray u-lineHeight--normal"> Restoring to this version will remove data and replace it with data from the restored version. During the restoration, your application will not be available and you will not be able to use the admin console. This action cannot be reversed. </p>
           </div>
           <div className="flex flex-column u-marginTop--20">
-            <p className="u-fontSize--normal u-fontWeight--bold u-color--tuna u-lineHeight--normal"> Type your application slug to continue </p>
+            <p className="u-fontSize--normal u-fontWeight--bold u-color--tuna u-lineHeight--normal"> Type your application slug to continue</p>
+
+            <p className="u-fontSize--normal u-fontWeight--normal u-color--dustyGray u-lineHeight--normal">We require you enter a matching app slug name as an additional confirmation you wish to proceed.</p>
```
Comment (GraysonNull) [inline kotsadm/web/src/components/modals/RestoreSnapshotModal.jsx:40]: I'd like to make a copy change suggestion to better match how we've spoke about this in other similar scenarios like requiring app name to delete an app in vendor web. 

`To confirm that you want to delete this snapshot, please type it's slug in the input as it appears below.`

**Review by GraysonNull — CHANGES_REQUESTED** [review]

File: kotsadm/web/src/components/modals/RestoreSnapshotModal.jsx (hunk 1 above)
Comment (GraysonNull) [reply kotsadm/web/src/components/modals/RestoreSnapshotModal.jsx:40]: After that change is made i'm good merging this. Thanks for removing Storybook until we can have a larger conversation about it's implementation.

**Comment by platformfirstco** [comment]
@GraysonNull i've updated the copy!

**Comment by GraysonNull** [comment]
Great, thank you!. One other question, why is there four "remove storybook" commits? That feels...odd

**Comment by platformfirstco** [comment]
> Great, thank you!. One other question, why is there four "remove storybook" commits? That feels...odd

Sorry about that! Every commit i thought i removed storybook, there was something lingering. Would you like me to reopen a PR without those commits?

**Comment by GraysonNull** [comment]
Ah gotcha. No it's ok, maybe just squash before merging. I was just wanting to make sure there wasn't anything messed up with the git history

**Comment by abhiaiyer91** [comment]
@GraysonNull can you squash and merge when you get a chance?

**Pull request merged**